CONTRIBUTING.md

```diff
@@ -167,7 +167,7 @@ If you ever find yourself at a loss for what to do, or in need of mentorship or
 Ultimately, Bevy is a tool that's designed to help people make cool games.
 By using Bevy, you can help us catch bugs, prioritize new features, polish off the rough edges, and promote the project.
 
-If you need help, don't hesitate to ask for help on [GitHub Discussions](https://github.com/bevyengine/bevy/discussions), [Discord](https://discord.gg/bevy), or [reddit](https://www.reddit.com/r/bevy). Generally you should prefer asking questions as Github Discussions as they are more searchable.
+If you need help, don't hesitate to ask for help on [GitHub Discussions](https://github.com/bevyengine/bevy/discussions), [Discord](https://discord.gg/bevy), or [reddit](https://www.reddit.com/r/bevy). Generally you should prefer asking questions as [Github Discussions](https://github.com/bevyengine/bevy/discussions) as they are more searchable.
```

Comment:
Links are good :) ; we might want to use [link references](https://github.github.com/gfm/#link-reference-definitions) to avoid duplication.

Reply:
It would be great if you could add them!

Reply:
_GitHub_ isn't spelled correctly here (capital H is missing)

Reply:
I added references for GitHub Discussion and Discord, and put them arbitrarily at the end of the file.

Reply:
Might as well do the same with the [reddit] link to remove a few duplaces of that one?